test/fake_package/test_readme.md

```diff
@@ -0,0 +1 @@
+Welcome to the Dart API reference documentation.
```

Comment:
Maybe include a bit of markdown syntax here so we can verify it's getting converted correctly?